python/lsst/ip/diffim/modelPsfMatch.py

```diff
@@ -372,15 +394,42 @@ def _buildCellSet(self, exposure, referencePsfModel):
                 widthList.append(widthS)
                 heightList.append(heightS)
 
-        lenMax = max(max(heightList), max(widthList))
-        lenMin = min(min(heightList), min(widthList))
+        psfSize = max(max(heightList), max(widthList))
 
-        lenPsfScience = lenMax if self.config.padPsf else lenMin
-        dimenS = afwGeom.Extent2I(lenPsfScience, lenPsfScience)
+        if self.config.doAutoPadPsf:
+            minPsfSize = nextOddInteger(self.kConfig.kernelSize*self.config.autoPadPsfTo)
```

Comment:
It may not be necessary, as this function is only called a small number of times, but it is possible to avoid all the overhead of calling a function, and the steps in the function by saying:
`minPsfSize = int(np.ceil(self.KConfig.kernelSize*self.config.autoPadPsfTo))|1`
This will return the number if it is odd, or the next integer if it is even